Documentation/Settings.cfg

```
@@ -0,0 +1,96 @@
+# #####
+#
+# Settings.cfg - A TYPO3 Documentation Project's Configuration File
+# Information about Settings.cfg:
+#   https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/GeneralConventions/DirectoryFilenames.html#settings-cfg
+#
+# About Syntax:
+#   See https://docs.python.org/2/library/configparser.html
+#
+# Attention:
+#   Only " ;" can start an inline comment.
+#   This is: blank PLUS semicolon!
+#
+# #####
+
+[general]
+
+# .................................................................................
+# ...   (required) title (displayed in left sidebar (desktop) or top panel (mobile)
+# .................................................................................
+
+project     = TYPO3 EXT:headless
+
+# .................................................................................
+# ...   (recommended) version, displayed next to title (desktop) and in <meta name="book-version"
+# .................................................................................
+
+release     = 2.x.x
+
+# .................................................................................
+# ...  (recommended) displayed in footer
+# .................................................................................
+
+copyright   = by TYPO3 EXT:headless
+
+[html_theme_options]
+
+# .................................................................................
+# ...  (recommended) to get the "Edit me on Github Button"
+# .................................................................................
+
+#github_branch        = master
+#github_repository    = TYPO3-Initiatives/headless
```

```suggestion
github_branch        = master
github_repository    = TYPO3-Initiatives/headless
```
Once this documentations has been published, these lines should not be commented

---

changed with commit be28db3490460d84eb6aefc6d96a12936f2a6c23